app/src/main/res/values/ids.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
nit: Please add a license header comment (see other XML files for examples)

Reply:
Added